Resources/Misc/emsg_list_detailed.txt

```diff
@@ -11,7 +11,6 @@
 	k_EMsgClientVACResponse = 704, // flags: 2, server type: -1
 	k_EMsgClientLogOff = 706, // flags: 2, server type: -1
 	k_EMsgClientNoUDPConnectivity = 707, // flags: 2, server type: -1
-	k_EMsgClientInformOfCreateAccount = 708, // flags: 2, server type: -1

```

Comment:
Do we usually remove entries from emsg_list_detailed?

Reply:
You did c817571caf36233e36931e13c3b878903e206219 e1f0549afc529c74646b600f57907d93e37ca731